.goreleaser.yml

```diff
@@ -14,7 +14,7 @@ builds:
 
   env:
     - CGO_ENABLED=0
-    
+
```

Comment:
remove the extra lines, lint is complaining about these.

Reply:
Actually, these are not newlines. My code editor automatically removed some trailing space these lines had.

On running lint checks locally, looks like the lint is complaining about some other files.

```
➜ golangci-lint run      
internal/graphql/resolver/addons.go:58: unnecessary trailing newline (whitespace)

	}()
internal/graphql/resolver/addons.go:56: unnecessary trailing newline (whitespace)

		}
internal/graphql/resolver/control-plane.go:56: unnecessary trailing newline (whitespace)

			}(ctxID)
```

PS: I don't think golangci-lint even checks `yaml` files.

Reply:
Made a pull request to fix these problems: https://github.com/meshery/meshery/pull/5890